packages/ui/src/styles/globals.css

```
@@ -543,11 +500,12 @@
   }
   body {
     @apply bg-background text-foreground;
+    overscroll-behavior: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)